_data/projects/OpenTemplate.yml

```
@@ -0,0 +1,8 @@
+name: open template
+desc: boilerplate for a new open source idea
+site: https://github.com/opentemplate/opentemplate
+tags:
+  - boilerplate
+upforgrabs:
+  name: up for grabs
```

:art: the CI error here is due to this label not being present - I think this one would work from your list, but there's no issues created on the project so I'm not 100% sure

```suggestion
  name: good first issue
```